Traceback (most recent call last):
  File "/home/nicola/Dokumente/Lehrbuch/mein_lehrbuch/my_jupyter_book/lib64/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/nicola/Dokumente/Lehrbuch/mein_lehrbuch/my_jupyter_book/lib64/python3.12/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nicola/Dokumente/Lehrbuch/mein_lehrbuch/my_jupyter_book/lib64/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/nicola/Dokumente/Lehrbuch/mein_lehrbuch/my_jupyter_book/lib64/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/nicola/Dokumente/Lehrbuch/mein_lehrbuch/my_jupyter_book/lib64/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/nicola/Dokumente/Lehrbuch/mein_lehrbuch/my_jupyter_book/lib64/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from sympy import symbols, cos, sin, pi, summation
t, omega_1, nu = symbols('t omega_1 nu')
a_nu, b_nu = symbols('a_nu b_nu')
x_t = a_nu/2 + summation(a_nu*cos(nu*omega_1*t) + b_nu*sin(nu*omega_1*t), (nu, 1, 10))
x_t
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01msympy[39;00m [38;5;28;01mimport[39;00m symbols, cos, sin, pi, summation
[1;32m      2[0m t, omega_1, nu [38;5;241m=[39m symbols([38;5;124m'[39m[38;5;124mt omega_1 nu[39m[38;5;124m'[39m)
[1;32m      3[0m a_nu, b_nu [38;5;241m=[39m symbols([38;5;124m'[39m[38;5;124ma_nu b_nu[39m[38;5;124m'[39m)

[0;31mModuleNotFoundError[0m: No module named 'sympy'

